nibabel/orientations.py

```diff
@@ -349,6 +349,9 @@ def axcodes2ornt(axcodes, labels=None):
            [ 0., -1.],
            [ 2.,  1.]])
     """
+
+    axcodes = tuple(c.upper() for c in axcodes)
```

Comment:
This is breaking on `labels is not None`. Probably the least disruptive approach is to move this into the `if labels is None` branch immediately below.

My reasoning is that, if somebody's passing labels, they'll also be passing axis codes with labels selected from that set. (The alternative would be to also apply `.upper()` to the labels, but I don't like the smell of that.)